cmd/importer/main.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"go.uber.org/zap/zapcore"
+	"k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	"sigs.k8s.io/kueue/cmd/importer/pod"
+	"sigs.k8s.io/kueue/cmd/importer/util"
+	"sigs.k8s.io/kueue/pkg/util/useragent"
+)
+
+const (
+	NamespaceFlag      = "namespace"
+	NamespaceFlagShort = "n"
+	QueueMappingFlag   = "queuemapping"
+	QueueLabelFlag     = "queuelabel"
+	QPSFlag            = "qps"
+	BurstFlag          = "burst"
+	VerbosityFlag      = "verbose"
+	VerboseFlagShort   = "v"
+	JobsFlag           = "jobs"
+	JobsFlagShort      = "j"
+)
+
+var (
+	k8sClient client.Client
+	rootCmd   = &cobra.Command{
+		Use:   "importer",
+		Short: "Import existing (running) objects into Kueue",
+		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
+			v, _ := cmd.Flags().GetCount(VerbosityFlag)
```

Comment:
Instead, can you use `zap.Options`'s `BindFlags`?
That way we get all the possible settings.

Reply:
I don't know how that works with cobra and we don't need all that functionality.